sonar-project.properties

```diff
@@ -1,3 +1,11 @@
 sonar.projectKey=vscode-didact
 sonar.projectName=VS Code Didact
-sonar.exclusions=**/*.java
\ No newline at end of file
+sonar.exclusions=**/*.java
+sonar.sources=src
+sonar.tests=src/test
+sonar.typescript.lcov.reportPaths=coverage/lcov.info
+
+sonar.links.homepage=https://github.com/redhat-developer/vscode-didact
+sonar.links.ci=https://app.circleci.com/pipelines/github/redhat-developer/vscode-didact
+sonar.links.scm=https://github.com/redhat-developer/vscode-didact
+sonar.links.issue=https://github.com/redhat-developer/vscode-didact/issues
```

Comment:
better to do it in a separate PR to not mix purpose in the same PR

Reply:
spun it off in https://github.com/redhat-developer/vscode-didact/pull/286